charts/stable/transmission/values.yaml

```diff
@@ -5,19 +5,21 @@
 # https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
 #
 
-# # Use an initContainer to download a custom web ui
-# initContainers:
-# - name: custom-webui
-#   image: curlimages/curl:7.76.1
-#   command:
-#   - "/bin/sh"
-#   - "-c"
-#   - "curl -fsSL -o /tmp/flood-for-transmission.tar.gz https://github.com/johman10/flood-for-transmission/releases/download/latest/flood-for-transmission.tar.gz"
-#   - "mkdir -p /config/flood"
-#   - "tar xf /tmp/flood-for-transmission.tar.gz --strip-components 1 -C /config/flood"
-#   volumeMounts:
-#   - name: config
-#     mountPath: /config
+# # Use an initContainer to download the Flood web ui
+# # Set UI with env TRANSMISSION_WEB_HOME: /config/flood-for-transmission/
+#initContainers:
+#- name: custom-webui
+#  image: curlimages/curl:7.76.1
+#  command:
+#  - "/bin/sh"
+#  - "-c"
+#  - "curl -o- -sL https://github.com/johman10/flood-for-transmission/releases/download/latest/flood-for-transmission.tar.gz | tar xzf - -C /config"
+#  volumeMounts:
+#  - name: config
+#    mountPath: /config
+#  securityContext:
+#    runAsUser: 568
+#    runAsGroup: 568
```

Comment:
```suggestion
#initContainers:
# - name: custom-webui
#   image: curlimages/curl:7.76.1
#   command:
#   - "/bin/sh"
#   - "-c"
#   - "curl -o- -sL https://github.com/johman10/flood-for-transmission/releases/download/latest/flood-for-transmission.tar.gz | tar xzf - -C /config"
#   volumeMounts:
#   - name: config
#     mountPath: /config
#   securityContext:
#     runAsUser: 568
#     runAsGroup: 568
```

Reply:
This should fix the linting issues hopefully

Reply:
> This should fix the linting issues hopefully

ah didn't even think about that after commenting out after testing

Reply:
I think it's still messed up. You should uncomment this and then let vscode comment the lines for you.